### PR TITLE
whitelist toolz

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist toolz

### Linked issues
Progresses #1901

### Functionality
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -26274,6 +26274,25 @@
   "tomlkit": {
     "tomlkit": []
   },
+  "toolz": {
+    "tlz": [],
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:26278]: Same question about the `tlz` not matching `toolz` as in other Pr

**Review by JCZuurmond — COMMENTED** [review]
See comment

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:26278]: We don't control this. The pypi package is `toolz`, it contains a `tlz` module, see https://github.com/pytoolz/toolz/

Comment (JCZuurmond) [reply src/databricks/labs/ucx/source_code/known.json]: yes, I understand that, but want to confirm that our logic will still work when this is the case

Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: this was tested by running `make solacc` so I confirm

**Review by JCZuurmond — APPROVED** [review]

**Pull request merged**